### PR TITLE
Release notes for v2.21.0

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

File: Release Notes/Release Notes v2.21.0.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# .NET Driver Version 2.21.0 Release Notes
+
+This is the general availability release for the 2.21.0 version of the driver.
+
+The main new features in 2.21.0 include:
+
+* LINQ3 bug fixes and improvements
+* Altas search sort, tracking, score details options added
+* Altas search embeddedDocuments operator support
+* Altas search index mangement API added
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.21.0.md:10]: Altas => Atlas

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Fix 3 typos noted then LGTM.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Review by rstam — APPROVED** [review]
LGTM

**Pull request merged**